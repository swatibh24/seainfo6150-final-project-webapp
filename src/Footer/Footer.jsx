import React from "react";
import styles from "./Footer.module.css";

const Footer = () => {
    return (
        <div className={styles.footer}> 
            <img  className={styles.logo}  src="http://shoppopbox.com/wp-content/uploads/2017/09/homechef-logo-square-1.jpg"
              alt="recipes logo" />
            <p className={styles.text}> Â©{new Date().getFullYear()} Home Made Recipes by Swati Bhojwani.</p>
        </div>
    )
};

export default Footer;