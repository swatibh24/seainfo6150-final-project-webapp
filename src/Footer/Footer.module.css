.logo{
  border-radius: 25%;
}

.text {
    font-family: Futura, Arial, sans-serif;
    color: black;
    text-align: center;
  }

.footer{
    margin-top: 1rem;
    padding: 1rem;
    background-color:#F4F4F4;
    position: relative;
    bottom: 0;
    left: 0;
    text-align: center;
}

  @media screen and (min-width: 400px) {
    .logo {
      width: 30px;
    }
    .title {
      font-size: 12px;
    }
    .navTab {
      margin-left: 20px;
      font-size: 15px;
    }
    .subscribe {
      padding: 1px 5px;
      font-size: 12px;
    }
  }

  @media screen and (min-width: 600px) {
    .logo {
      width: 40px;
    }
    .title {
      font-size: 15px;
    }
    .navTab {
      margin-left: 20px;
      font-size: 15px;
    }
    .subscribe {
      padding: 2px 10px;
      font-size: 15px;
    }
  }
  
  @media screen and (min-width: 900px) {
    .logo {
      width: 60px;
    }
    .title {
      font-size: 20px;
    }
    .navTab {
      margin-left: 50px;
      font-size: 18px;
    }
    .subscribe {
      padding: 5px 20px;
      font-size: 18px;
    }
  }
  
  @media screen and (min-width: 1200px) {
    .logo {
      width: 80px;
    }
    .title {
      font-size: 25px;
    }
    .navTab {
      margin-left: 80px;
      font-size: 20px;
    }
    .subscribe {
      padding: 5px 25px;
      font-size: 20px;
    }
  }
  