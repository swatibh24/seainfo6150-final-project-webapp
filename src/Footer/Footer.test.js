import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer.jsx";


describe("Register tests", () => {
    it("renders correctly", () => {
      const { container } = render(<Footer />);
      expect(container).toMatchSnapshot();
    });
  });