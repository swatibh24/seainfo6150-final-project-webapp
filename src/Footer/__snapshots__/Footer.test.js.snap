// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register tests renders correctly 1`] = `
<div>
  <div
    class="footer"
  >
    <img
      alt="recipes logo"
      class="logo"
      src="http://shoppopbox.com/wp-content/uploads/2017/09/homechef-logo-square-1.jpg"
    />
    <p
      class="text"
    >
       Â©
      2021
       Home Made Recipes by Swati Bhojwani.
    </p>
  </div>
</div>
`;
